notebooks/demo2/time-anlysis.ipynb

```diff
@@ -0,0 +1,1187 @@
+{
```

Comment:
<p>Just checking if this is correct, as i can see from the above statistical data. 75% of pdfs have pages in the range 1 to 224.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/os-climate/aicoe-osc-demo/pull/217/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
<p>Updated :)</p>